src/Components/Shared/src/ResourceCollectionProvider.cs

```diff
@@ -11,36 +11,29 @@ namespace Microsoft.AspNetCore.Components;
 
 internal class ResourceCollectionProvider
```

Comment:
We need to register the provider as persistent component state service, right?

Reply:
I agree, it's done with. Isn't?
```
RegisterPersistentComponentStateServiceCollectionExtensions.AddPersistentServiceRegistration<ResourceCollectionProvider>(services, RenderMode.InteractiveWebAssembly);
```

Reply:
Oh, the line was too long, and I didn't scroll to the end :D
My bad, sorry